### ISSUE TITLE
MSE (MeanSquaredError) loss function

### ISSUE THREAD

**Issue by MopsieX** [SciSharp/TensorFlow.NET#651]
MSE would be a great addition to have.

**Comment by Oceania2018** [comment]
We will add unit test as this [docs](https://keras.io/api/losses/regression_losses/#meansquarederror-class) for `MeanSquaredError`.

**Comment by MopsieX** [comment]
Awesome thanks so much!
